### ISSUE TITLE
Research APIs for creating holdings records

### ISSUE THREAD

**Issue by lwrubel** [sul-dlss/folio_client#33]
This is in support of adding 856 fields to records being released/published.

**Comment by shelleydoljack** [comment]
You can get the different holdings types with [GET /holdings-types](https://s3.amazonaws.com/foliodocs/api/mod-inventory-storage/p/holdings-type.html). I guess "Electronic" is the one you will always want for this, so 👍 . With the location ID, again, you can do [GET /locations](https://s3.amazonaws.com/foliodocs/api/mod-inventory-storage/p/location.html). We have at least 400 locations, but you probably only want the locations where the primary service point is "Online" (`${OKAPI_URL}/locations?query=primaryServicePoint==7c5abc9f-f3d7-4856-b8d7-6712462ca007`). Although, maybe it'd be better to do a query for stuff with "SDR" or "ELECTRONIC" in the name or code, such as `${OKAPI_URL}/locations?query=cql.allRecords=1 and name any "ELECTRONIC SDR"`. @ahafele I noticed that the coordinate library ELECTRONIC locations do not have "Online" as the primary service point. Should it be? But maybe the cql query to the locations endpoint is more exact for the location ID's that are needed to update 856's.

**Comment by ahafele** [comment]
@shelleydoljack yes I think they should. I'll make a ticket to update these. 
 I think in this context we are only concerned with SDR locations. I don't think you need to even include Electronic.

**Comment by shelleydoljack** [comment]
It looks like we also need to make the LAW-SDR, etc. locations, right @ahafele ?

**Comment by ahafele** [comment]
Correct - we haven't done that work yet - https://github.com/sul-dlss/FOLIO-Project-Stanford/issues/252

**Comment by lwrubel** [comment]
For SDR, this looks like the location we'll want to use, based on a query for `'name any "SDR"'`:
```
{'locations': [{'id': '1b14e21c-8d47-45c7-bc49-456a0086422b',
   'name': 'SUL SDR',
   'code': 'SUL-SDR',
   'discoveryDisplayName': 'SUL SDR',
   'isActive': True,
   'institutionId': '8d433cdd-4e8f-4dc1-aa24-8a4ddb7dc929',
   'campusId': 'c365047a-51f2-45ce-8601-e421ca3615c5',
   'libraryId': 'c1a86906-ced0-46cb-8f5b-8cef542bdd00',
   'primaryServicePoint': '7c5abc9f-f3d7-4856-b8d7-6712462ca007',
   'servicePointIds': ['7c5abc9f-f3d7-4856-b8d7-6712462ca007'],
   'servicePoints': [],
   'metadata': {'createdDate': '2023-02-08T23:38:05.776+00:00',
    'createdByUserId': '7dc1ae38-1e06-4de9-ac86-568b9a2318ec',
    'updatedDate': '2023-02-08T23:38:05.776+00:00',
    'updatedByUserId': '7dc1ae38-1e06-4de9-ac86-568b9a2318ec'}}],
 'totalRecords': 1}
```

**Comment by lwrubel** [comment]
Closed with creation of implementation ticket #39.
